docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md

```diff
@@ -39,9 +39,9 @@ Click all the tick boxes, including the "Add a rectangle arena" which is not tic
 
 Webots displays a list of directories and files it just created.
 This corresponds to the [standard file hierarchy of a Webots project](the-standard-file-hierarchy-of-a-project.md).
-Click on the `Finish` button to close this window.
+Click on the `Done` button to close this window.
```

Comment:
I indeed get a `Finish` button on Windows. Could that be different on macOS?

Reply:
Click on the `Finish` (Windows, Linux) or `Done` (macOS) button to close this window.

Reply:
```suggestion
Click on the `Finish` (Windows, Linux) or `Done` (macOS) button to close this window.
```